static_report/src/hooks/useAmplitudeTrack.ts

```diff
@@ -0,0 +1,14 @@
+import { track } from '@amplitude/analytics-browser';
+import type { EventOptions } from '@amplitude/analytics-types';
+
+export type UserAmplitudeTrack = {
+  eventName: string;
+  eventProperties?: Record<string, unknown> | undefined;
```

Comment:
no need for `| undefined` as optional `?` with `strictNullCheck` will now infer correctly